### PR TITLE
Update to include sample workshop for first class

### PR DESCRIPTION
**Pull request by redixhumayun**
Let me know what you think. Sorry about the disgusting framework I uploaded on imgur, but did that in a hurry.

### CODE REVIEW

File: intro-html.md [hunk 1]
```diff
@@ -130,4 +130,13 @@ Has te zril voluptatum philosophia, cum velit graeci mnesarchum id, ex his nulla
 ```
 
 ## Workshop
-TODO
\ No newline at end of file
+TODO
+
+####Tribute Page
```
Comment (maneesht) [inline intro-html.md]: Add a space between the hashes and the title

File: intro-html.md [hunk 2]
```diff
@@ -130,4 +130,13 @@ Has te zril voluptatum philosophia, cum velit graeci mnesarchum id, ex his nulla
...
+For the very first workshop, create a simple tribute page to any personality of your choice. The basic requirements for this are that the web page have a navbar at the top, with the name of the person this is a tribute to and your name in the right of the navbar. There should be an image of the tributee, which should be a link that directs the user to a website where they can read more. A body of text that is the width of the image should follow containing information about the tributee. 
+
+A very basic wireframe of what it should look like can be found [here](http://imgur.com/a/zQvBy)
+
+A sample example can be found [here](https://codepen.io/redixhumayun/pen/pyGByG)
+
+Beyond these requirements,
```
Comment (maneesht) [inline intro-html.md]: Thought needs to be finished.

**Review by maneesht — CHANGES_REQUESTED** [review]
Overall, this looks good so far. The example given is somewhat complex because it uses bootstrap classes. This may confuse some students.

File: intro-html.md (hunk 1 above)
Comment (redixhumayun) [inline intro-html.md]: I can try to do the same without Bootstrap. Want me to go ahead and do that? I can host it and send you a link on codepen.io

**Comment by redixhumayun** [comment]
I can try and create a similar page without bootstrap. Want me to go ahead and do that? I can share on codepen.io later

**Comment by maneesht** [comment]
Can you do that?

File: intro-html.md [hunk 3]
```diff
@@ -130,4 +130,13 @@ Has te zril voluptatum philosophia, cum velit graeci mnesarchum id, ex his nulla
 ```
 
 ## Workshop
-TODO
\ No newline at end of file
+TODO
+
+####Tribute Page
+For the very first workshop, create a simple tribute page to any personality of your choice. The basic requirements for this are that the web page have a navbar at the top, with the name of the person this is a tribute to and your name in the right of the navbar. There should be an image of the tributee, which should be a link that directs the user to a website where they can read more. A body of text that is the width of the image should follow containing information about the tributee.
```
Comment (maneesht) [inline intro-html.md]: Not necessarily a nav bar, just a title section

**Comment by maneesht** [comment]
Also, don't forget to add very simple boilerplate code for the workshop

**Comment by maneesht** [comment]
Merged

**Pull request closed without merging**